src/server/scripts/Northrend/Naxxramas/boss_grobbulus.cpp

```diff
@@ -53,16 +69,16 @@ struct boss_grobbulus : public BossAI
     void JustEngagedWith(Unit* who) override
     {
         BossAI::JustEngagedWith(who);
-        events.ScheduleEvent(EVENT_CLOUD, 15s);
-        events.ScheduleEvent(EVENT_INJECT, 20s);
-        events.ScheduleEvent(EVENT_SPRAY, randtime(Seconds(15), Seconds(30))); // not sure
+        events.ScheduleEvent(EVENT_CLOUD, 5s, 10s);
+        events.ScheduleEvent(EVENT_INJECT, 12s, 15s);
+        events.ScheduleEvent(EVENT_SPRAY, RAND(25700ms, 27700ms, 30000ms));
         events.ScheduleEvent(EVENT_BERSERK, 12min);
     }
 
     void SpellHitTarget(WorldObject* target, SpellInfo const* spellInfo) override
     {
-        if (spellInfo->Id == SPELL_SLIME_SPRAY)
-            me->SummonCreature(NPC_FALLOUT_SLIME, *target, TEMPSUMMON_TIMED_DESPAWN_OUT_OF_COMBAT);
+        if (spellInfo->Id == SPELL_SLIME_SPRAY || spellInfo->Id == SPELL_SLIME_SPRAY_H)
```

Comment:
nice catch, people usually forget to check difficulty entries.
though I recall a helper method to fetch the normal entry for a spell in an instance, given the difficulty entry...